CHANGELOG.md

```diff
@@ -74,6 +76,26 @@ librdkafka.
 
 ### Producer fixes
 
+ * Transactional producer could crash on request timeout due to dereferencing
+   NULL pointer of non-existent response object (ESC-3009)
+ * Mark `rd_kafka_send_offsets_to_transaction()` CONCURRENT_TRANSACTION (et.al)
+   errors as retriable (ESC-2998)
+ * `socket.timeout.ms` was ignored when `transactional.id` was set.
+
+
+# librdkafka v1.4.4
+
+v1.4.4 is a maintenance release with the following fixes and enhancements:
+
+ * Transactional producer could crash on request timeout due to dereferencing
+   NULL pointer of non-existent response object (ESC-3009)
+ * Mark `rd_kafka_send_offsets_to_transaction()` CONCURRENT_TRANSACTION (et.al)
+   errors as retriable (ESC-2998)
+ * Fix theoeretical crash on coord request failure.
```

Comment:
what is a theoretical crash?

Reply:
I effed something else up which triggered this crash, but we havent seen any reports of it in real life.